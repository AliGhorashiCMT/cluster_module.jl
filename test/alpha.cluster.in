ion-species GBRV/$ID_pbesol.uspp 
coords-type Cartesian
ion	O	3.3405371999999995	-1.2443517191191242	2.313767999995583	1	
ion	O	8.2545372	-9.755649387512584	2.313767999995583	1	
ion	O	13.1685372	-18.266947055906048	2.313767999995583	1	
ion	O	8.254537199999998	7.266945949274335	2.313767999995583	1	
ion	O	13.1685372	-1.2443517191191251	2.313767999995583	1	
ion	O	18.082537199999997	-9.755649387512587	2.313767999995583	1	
ion	O	13.168537199999998	15.778243617667796	2.313767999995583	1	
ion	O	18.082537199999997	7.266945949274335	2.313767999995583	1	
ion	O	22.9965372	-1.244351719119127	2.313767999995583	1	
ion	O	3.3405371999999995	-1.2443517191191242	13.125767999974942	1	
ion	O	8.2545372	-9.755649387512584	13.125767999974942	1	
ion	O	13.1685372	-18.266947055906048	13.125767999974942	1	
ion	O	8.254537199999998	7.266945949274335	13.125767999974942	1	
ion	O	13.1685372	-1.2443517191191251	13.125767999974942	1	
ion	O	18.082537199999997	-9.755649387512587	13.125767999974942	1	
ion	O	13.168537199999998	15.778243617667796	13.125767999974942	1	
ion	O	18.082537199999997	7.266945949274335	13.125767999974942	1	
ion	O	22.9965372	-1.244351719119127	13.125767999974942	1	
ion	O	4.321371599999999	-4.996131731346962	5.917764395988702	1	
ion	O	9.235371599999999	-13.507429399740422	5.917764395988702	1	
ion	O	14.149371599999997	-22.01872706813388	5.917764395988702	1	
ion	O	9.235371599999999	3.5151659370464987	5.917764395988702	1	
ion	O	14.149371599999998	-4.996131731346962	5.917764395988702	1	
ion	O	19.063371599999996	-13.50742939974042	5.917764395988702	1	
ion	O	14.149371599999998	12.02646360543996	5.917764395988702	1	
ion	O	19.063371599999996	3.5151659370464987	5.917764395988702	1	
ion	O	23.977371599999998	-4.99613173134696	5.917764395988702	1	
ion	O	4.321371599999999	-4.996131731346962	16.729764395968065	1	
ion	O	9.235371599999999	-13.507429399740422	16.729764395968065	1	
ion	O	14.149371599999997	-22.01872706813388	16.729764395968065	1	
ion	O	9.235371599999999	3.5151659370464987	16.729764395968065	1	
ion	O	14.149371599999998	-4.996131731346962	16.729764395968065	1	
ion	O	19.063371599999996	-13.50742939974042	16.729764395968065	1	
ion	O	14.149371599999998	12.02646360543996	16.729764395968065	1	
ion	O	19.063371599999996	3.5151659370464987	16.729764395968065	1	
ion	O	23.977371599999998	-4.99613173134696	16.729764395968065	1	
ion	O	7.0800912	-2.2708142179273763	9.521771603981822	1	
ion	O	11.9940912	-10.782111886320838	9.521771603981822	1	
ion	O	16.908091199999998	-19.293409554714298	9.521771603981822	1	
ion	O	11.994091199999998	6.240483450466083	9.521771603981822	1	
ion	O	16.908091199999998	-2.270814217927377	9.521771603981822	1	
ion	O	21.822091199999996	-10.782111886320838	9.521771603981822	1	
ion	O	16.9080912	14.751781118859544	9.521771603981822	1	
ion	O	21.8220912	6.240483450466083	9.521771603981822	1	
ion	O	26.736091199999997	-2.270814217927377	9.521771603981822	1	
ion	O	7.0800912	-2.2708142179273763	20.333771603961182	1	
ion	O	11.9940912	-10.782111886320838	20.333771603961182	1	
ion	O	16.908091199999998	-19.293409554714298	20.333771603961182	1	
ion	O	11.994091199999998	6.240483450466083	20.333771603961182	1	
ion	O	16.908091199999998	-2.270814217927377	20.333771603961182	1	
ion	O	21.822091199999996	-10.782111886320838	20.333771603961182	1	
ion	O	16.9080912	14.751781118859544	20.333771603961182	1	
ion	O	21.8220912	6.240483450466083	20.333771603961182	1	
ion	O	26.736091199999997	-2.270814217927377	20.333771603961182	1	
ion	O	3.3405371999999995	1.2443517191191242	8.498231999983778	1	
ion	O	8.254537199999998	-7.266945949274335	8.498231999983778	1	
ion	O	13.168537199999998	-15.778243617667796	8.498231999983778	1	
ion	O	8.2545372	9.755649387512584	8.498231999983778	1	
ion	O	13.1685372	1.2443517191191251	8.498231999983778	1	
ion	O	18.082537199999997	-7.266945949274335	8.498231999983778	1	
ion	O	13.1685372	18.266947055906048	8.498231999983778	1	
ion	O	18.082537199999997	9.755649387512587	8.498231999983778	1	
ion	O	22.9965372	1.244351719119127	8.498231999983778	1	
ion	O	3.3405371999999995	1.2443517191191242	19.310231999963136	1	
ion	O	8.254537199999998	-7.266945949274335	19.310231999963136	1	
ion	O	13.168537199999998	-15.778243617667796	19.310231999963136	1	
ion	O	8.2545372	9.755649387512584	19.310231999963136	1	
ion	O	13.1685372	1.2443517191191251	19.310231999963136	1	
ion	O	18.082537199999997	-7.266945949274335	19.310231999963136	1	
ion	O	13.1685372	18.266947055906048	19.310231999963136	1	
ion	O	18.082537199999997	9.755649387512587	19.310231999963136	1	
ion	O	22.9965372	1.244351719119127	19.310231999963136	1	
ion	O	4.321371599999999	4.996131731346962	4.894235603990657	1	
ion	O	9.235371599999999	-3.5151659370464987	4.894235603990657	1	
ion	O	14.149371599999998	-12.02646360543996	4.894235603990657	1	
ion	O	9.235371599999999	13.507429399740422	4.894235603990657	1	
ion	O	14.149371599999998	4.996131731346962	4.894235603990657	1	
ion	O	19.063371599999996	-3.5151659370464987	4.894235603990657	1	
ion	O	14.149371599999997	22.01872706813388	4.894235603990657	1	
ion	O	19.063371599999996	13.50742939974042	4.894235603990657	1	
ion	O	23.977371599999998	4.99613173134696	4.894235603990657	1	
ion	O	4.321371599999999	4.996131731346962	15.706235603970017	1	
ion	O	9.235371599999999	-3.5151659370464987	15.706235603970017	1	
ion	O	14.149371599999998	-12.02646360543996	15.706235603970017	1	
ion	O	9.235371599999999	13.507429399740422	15.706235603970017	1	
ion	O	14.149371599999998	4.996131731346962	15.706235603970017	1	
ion	O	19.063371599999996	-3.5151659370464987	15.706235603970017	1	
ion	O	14.149371599999997	22.01872706813388	15.706235603970017	1	
ion	O	19.063371599999996	13.50742939974042	15.706235603970017	1	
ion	O	23.977371599999998	4.99613173134696	15.706235603970017	1	
ion	O	7.0800912	2.2708142179273763	1.290228395997537	1	
ion	O	11.994091199999998	-6.240483450466083	1.290228395997537	1	
ion	O	16.9080912	-14.751781118859544	1.290228395997537	1	
ion	O	11.9940912	10.782111886320838	1.290228395997537	1	
ion	O	16.908091199999998	2.270814217927377	1.290228395997537	1	
ion	O	21.8220912	-6.240483450466083	1.290228395997537	1	
ion	O	16.908091199999998	19.293409554714298	1.290228395997537	1	
ion	O	21.822091199999996	10.782111886320838	1.290228395997537	1	
ion	O	26.736091199999997	2.270814217927377	1.290228395997537	1	
ion	O	7.0800912	2.2708142179273763	12.102228395976896	1	
ion	O	11.994091199999998	-6.240483450466083	12.102228395976896	1	
ion	O	16.9080912	-14.751781118859544	12.102228395976896	1	
ion	O	11.9940912	10.782111886320838	12.102228395976896	1	
ion	O	16.908091199999998	2.270814217927377	12.102228395976896	1	
ion	O	21.8220912	-6.240483450466083	12.102228395976896	1	
ion	O	16.908091199999998	19.293409554714298	12.102228395976896	1	
ion	O	21.822091199999996	10.782111886320838	12.102228395976896	1	
ion	O	26.736091199999997	2.270814217927377	12.102228395976896	1	
ion	Si	2.3090885999999995	-3.9994587743780867	3.60399639599312	1	
ion	Si	7.2230886	-12.510756442771548	3.60399639599312	1	
ion	Si	12.137088599999998	-21.022054111165005	3.60399639599312	1	
ion	Si	7.223088599999999	4.511838894015374	3.60399639599312	1	
ion	Si	12.137088599999998	-3.9994587743780876	3.60399639599312	1	
ion	Si	17.0510886	-12.510756442771545	3.60399639599312	1	
ion	Si	12.137088599999998	13.023136562408833	3.60399639599312	1	
ion	Si	17.0510886	4.511838894015373	3.60399639599312	1	
ion	Si	21.965088599999998	-3.999458774378084	3.60399639599312	1	
ion	Si	2.3090885999999995	-3.9994587743780867	14.415996395972481	1	
ion	Si	7.2230886	-12.510756442771548	14.415996395972481	1	
ion	Si	12.137088599999998	-21.022054111165005	14.415996395972481	1	
ion	Si	7.223088599999999	4.511838894015374	14.415996395972481	1	
ion	Si	12.137088599999998	-3.9994587743780876	14.415996395972481	1	
ion	Si	17.0510886	-12.510756442771545	14.415996395972481	1	
ion	Si	12.137088599999998	13.023136562408833	14.415996395972481	1	
ion	Si	17.0510886	4.511838894015373	14.415996395972481	1	
ion	Si	21.965088599999998	-3.999458774378084	14.415996395972481	1	
ion	Si	2.3090885999999995	3.9994587743780867	7.20800360398624	1	
ion	Si	7.223088599999999	-4.511838894015374	7.20800360398624	1	
ion	Si	12.137088599999998	-13.023136562408833	7.20800360398624	1	
ion	Si	7.2230886	12.510756442771548	7.20800360398624	1	
ion	Si	12.137088599999998	3.9994587743780876	7.20800360398624	1	
ion	Si	17.0510886	-4.511838894015373	7.20800360398624	1	
ion	Si	12.137088599999998	21.022054111165005	7.20800360398624	1	
ion	Si	17.0510886	12.510756442771545	7.20800360398624	1	
ion	Si	21.965088599999998	3.999458774378084	7.20800360398624	1	
ion	Si	2.3090885999999995	3.9994587743780867	18.0200036039656	1	
ion	Si	7.223088599999999	-4.511838894015374	18.0200036039656	1	
ion	Si	12.137088599999998	-13.023136562408833	18.0200036039656	1	
ion	Si	7.2230886	12.510756442771548	18.0200036039656	1	
ion	Si	12.137088599999998	3.9994587743780876	18.0200036039656	1	
ion	Si	17.0510886	-4.511838894015373	18.0200036039656	1	
ion	Si	12.137088599999998	21.022054111165005	18.0200036039656	1	
ion	Si	17.0510886	12.510756442771545	18.0200036039656	1	
ion	Si	21.965088599999998	3.999458774378084	18.0200036039656	1	
ion	Si	5.2098227999999995	0.0	0.0	1	
ion	Si	10.1238228	-8.51129766839346	0.0	1	
ion	Si	15.037822799999997	-17.02259533678692	0.0	1	
ion	Si	10.1238228	8.51129766839346	0.0	1	
ion	Si	15.037822799999999	0.0	0.0	1	
ion	Si	19.9518228	-8.51129766839346	0.0	1	
ion	Si	15.037822799999997	17.02259533678692	0.0	1	
ion	Si	19.9518228	8.51129766839346	0.0	1	
ion	Si	24.865822799999997	0.0	0.0	1	
ion	Si	5.2098227999999995	0.0	10.81199999997936	1	
ion	Si	10.1238228	-8.51129766839346	10.81199999997936	1	
ion	Si	15.037822799999997	-17.02259533678692	10.81199999997936	1	
ion	Si	10.1238228	8.51129766839346	10.81199999997936	1	
ion	Si	15.037822799999999	0.0	10.81199999997936	1	
ion	Si	19.9518228	-8.51129766839346	10.81199999997936	1	
ion	Si	15.037822799999997	17.02259533678692	10.81199999997936	1	
ion	Si	19.9518228	8.51129766839346	10.81199999997936	1	
ion	Si	24.865822799999997	0.0	10.81199999997936	1	


lattice\ 
 50 0 0 \ 
 0 50 0 \ 
 0 0 50
