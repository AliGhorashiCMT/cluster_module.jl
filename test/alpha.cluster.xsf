CRYSTAL
PRIMVEC
 26.458860   0.000000   0.000000
  0.000000  26.458860   0.000000
  0.000000   0.000000  26.458860
PRIMCOORD
35 1
 8   4.368113   3.845502   1.224393
 8   4.368113   3.845502   6.945857
 8   6.968490   8.349487   6.945857
 8   9.568866   3.845502   6.945857
 8   4.887148   1.860146   3.131546
 8   7.487525   6.364130   3.131546
 8   4.887148   1.860146   8.853010
 8   7.487525   6.364130   8.853010
 8   6.347000   3.302322   5.038704
 8   8.947376   7.806306   5.038704
 8   4.368113   5.162467   4.497071
 8   6.968490   0.658483   4.497071
 8   9.568866   5.162467   4.497071
 8   2.286771   2.643839   2.589918
 8   4.887148   7.147824   2.589918
 8   7.487525   2.643839   2.589918
 8   2.286771   2.643839   8.311382
 8   4.887148   7.147824   8.311382
 8   7.487525   2.643839   8.311382
 8   6.347000   5.705648   0.682759
 8   3.746623   1.201663   6.404223
 8   6.347000   5.705648   6.404223
 8   8.947376   1.201663   6.404223
14   3.822294   2.387562   1.907153
14   6.422671   6.891547   1.907153
14   3.822294   2.387562   7.628617
14   6.422671   6.891547   7.628617
14   9.023047   2.387562   7.628617
14   1.221917   2.116422   3.814311
14   3.822294   6.620407   3.814311
14   6.422671   2.116422   3.814311
14   9.023047   6.620407   3.814311
14   5.357296   4.503985   5.721464
14   7.957673   9.007969   5.721464
14  10.558050   4.503985   5.721464
