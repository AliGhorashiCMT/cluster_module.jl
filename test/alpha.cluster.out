
*************** JDFTx 1.6.0  ***************

Start date and time: Sat Jun  5 22:59:33 2021
Executable jdftx with command-line: -ni alpha.cluster.in
Running on hosts (process indices):  ALIs-MacBook-Pro.local (0)
Divided in process groups (process indices):  0 (0)
Resource initialization completed at t[s]:      0.00
Run totals: 1 processes, 8 threads, 0 GPUs


Input parsed successfully to the following command list (including defaults):

basis kpoint-dependent
coords-type Cartesian
core-overlap-check vector
coulomb-interaction Periodic
davidson-band-ratio 1.1
dump End State
dump-name $INPUT.$VAR
elec-cutoff 20
elec-eigen-algo Davidson
elec-ex-corr gga-PBE
electronic-minimize  \
	dirUpdateScheme      FletcherReeves \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  1e-08 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
exchange-regularization WignerSeitzTruncated
fluid None
fluid-ex-corr  lda-PZ
fluid-gummel-loop 10 1.000000e-05
fluid-minimize  \
	dirUpdateScheme      PolakRibiere \
	linminMethod         DirUpdateRecommended \
	nIterations          100 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  0 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
fluid-solvent H2O 55.338 ScalarEOS \
	epsBulk 78.4 \
	pMol 0.92466 \
	epsInf 1.77 \
	Pvap 1.06736e-10 \
	sigmaBulk 4.62e-05 \
	Rvdw 2.61727 \
	Res 1.42 \
	tauNuc 343133 \
	poleEl 15 7 1
forces-output-coords Positions
ion O   8.254537199999998   7.266945949274335   2.313767999995583 1
ion O   8.254537199999998   7.266945949274335  13.125767999974942 1
ion O  13.168537199999996  15.778243617667794  13.125767999974942 1
ion O  18.082537199999997   7.266945949274335  13.125767999974942 1
ion O   9.235371599999999   3.515165937046499   5.917764395988702 1
ion O  14.149371599999998  12.026463605439959   5.917764395988702 1
ion O   9.235371599999999   3.515165937046499  16.729764395968065 1
ion O  14.149371599999998  12.026463605439959  16.729764395968065 1
ion O  11.994091199999998   6.240483450466083   9.521771603981822 1
ion O  16.908091200000001  14.751781118859544   9.521771603981822 1
ion O   8.254537200000000   9.755649387512584   8.498231999983778 1
ion O  13.168537199999999   1.244351719119125   8.498231999983778 1
ion O  18.082537199999997   9.755649387512587   8.498231999983778 1
ion O   4.321371599999999   4.996131731346962   4.894235603990657 1
ion O   9.235371599999999  13.507429399740422   4.894235603990657 1
ion O  14.149371599999998   4.996131731346962   4.894235603990657 1
ion O   4.321371599999999   4.996131731346962  15.706235603970017 1
ion O   9.235371599999999  13.507429399740422  15.706235603970017 1
ion O  14.149371599999998   4.996131731346962  15.706235603970017 1
ion O  11.994091200000000  10.782111886320838   1.290228395997537 1
ion O   7.080091200000001   2.270814217927376  12.102228395976896 1
ion O  11.994091200000000  10.782111886320838  12.102228395976896 1
ion O  16.908091199999998   2.270814217927377  12.102228395976896 1
ion Si   7.223088600000000   4.511838894015374   3.603996395993120 1
ion Si  12.137088599999998  13.023136562408833   3.603996395993120 1
ion Si   7.223088600000000   4.511838894015374  14.415996395972483 1
ion Si  12.137088599999998  13.023136562408833  14.415996395972483 1
ion Si  17.051088600000000   4.511838894015373  14.415996395972483 1
ion Si   2.309088599999999   3.999458774378087   7.208003603986240 1
ion Si   7.223088600000000  12.510756442771548   7.208003603986240 1
ion Si  12.137088599999998   3.999458774378088   7.208003603986240 1
ion Si  17.051088600000000  12.510756442771545   7.208003603986240 1
ion Si  10.123822799999999   8.511297668393460  10.811999999979360 1
ion Si  15.037822799999997  17.022595336786921  10.811999999979360 1
ion Si  19.951822799999999   8.511297668393460  10.811999999979360 1
ion-species GBRV/$ID_pbesol.uspp
ion-width 0
ionic-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0.0001 \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
kpoint   0.000000000000   0.000000000000   0.000000000000  1.00000000000000
kpoint-folding 1 1 1 
latt-move-scale 1 1 1
latt-scale 1 1 1 
lattice  \
	  50.000000000000000    0.000000000000000    0.000000000000000  \
	   0.000000000000000   50.000000000000000    0.000000000000000  \
	   0.000000000000000    0.000000000000000   50.000000000000000 
lattice-minimize  \
	dirUpdateScheme      L-BFGS \
	linminMethod         DirUpdateRecommended \
	nIterations          0 \
	history              15 \
	knormThreshold       0 \
	energyDiffThreshold  1e-06 \
	nEnergyDiff          2 \
	alphaTstart          1 \
	alphaTmin            1e-10 \
	updateTestStepSize   yes \
	alphaTreduceFactor   0.1 \
	alphaTincreaseFactor 3 \
	nAlphaAdjustMax      3 \
	wolfeEnergy          0.0001 \
	wolfeGradient        0.9 \
	fdTest               no
lcao-params -1 1e-06 0.001
pcm-variant GLSSA13
spintype no-spin
subspace-rotation-factor 1 yes
symmetries automatic
symmetry-threshold 0.0001



---------- Setting up symmetries ----------

Found 48 point-group symmetries of the bravais lattice
Found 1 space-group symmetries with basis
Applied RMS atom displacement 0 bohrs to make symmetries exact.

---------- Initializing the Grid ----------
R = 
[           50            0            0  ]
[            0           50            0  ]
[            0            0           50  ]
unit cell volume = 125000
G =
[   0.125664          0          0  ]
[          0   0.125664          0  ]
[          0          0   0.125664  ]
Minimum fftbox size, Smin = [  204  204  204  ]
Chosen fftbox size, S = [  210  210  210  ]

---------- Exchange Correlation functional ----------
Initalized PBE GGA exchange.
Initalized PBE GGA correlation.

---------- Setting up pseudopotentials ----------
Width of ionic core gaussian charges (only for fluid interactions / plotting) set to 0

Reading pseudopotential file '/Users/ali/Desktop/JDFTX/build-withlibxc/pseudopotentials/GBRV/o_pbesol.uspp':
  Title: O.  Created by USPP 7.3.6 on 3-2-2014
  Reference state energy: -15.849885.  6 valence electrons in orbitals:
    |200>  occupation: 2  eigenvalue: -0.873214
    |210>  occupation: 4  eigenvalue: -0.330058
  lMax: 2  lLocal: 2  QijEcut: 6
  5 projectors sampled on a log grid with 511 points:
    l: 0  eig: -0.873214  rCut: 1.25
    l: 0  eig: 0.000000  rCut: 1.25
    l: 1  eig: -0.330058  rCut: 1.25
    l: 1  eig: 0.000000  rCut: 1.25
    l: 2  eig: 1.000000  rCut: 1.25
  Partial core density with radius 0.7
  Transforming core density to a uniform radial grid of dG=0.02 with 1148 points.
  Transforming local potential to a uniform radial grid of dG=0.02 with 1148 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
  Transforming density augmentations to a uniform radial grid of dG=0.02 with 1148 points.
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 1.25 bohrs.

Reading pseudopotential file '/Users/ali/Desktop/JDFTX/build-withlibxc/pseudopotentials/GBRV/si_pbesol.uspp':
  Title: Si.  Created by USPP 7.3.6 on 23-7-2013
  Reference state energy: -4.568560.  4 valence electrons in orbitals:
    |300>  occupation: 2  eigenvalue: -0.395743
    |310>  occupation: 2  eigenvalue: -0.149098
  lMax: 2  lLocal: 3  QijEcut: 5
  6 projectors sampled on a log grid with 627 points:
    l: 0  eig: -0.395743  rCut: 1.6
    l: 0  eig: 1.000000  rCut: 1.6
    l: 1  eig: -0.149098  rCut: 1.6
    l: 1  eig: 1.000000  rCut: 1.6
    l: 2  eig: -0.100000  rCut: 1.7
    l: 2  eig: 0.100000  rCut: 1.7
  Partial core density with radius 1.45
  Transforming core density to a uniform radial grid of dG=0.02 with 1148 points.
  Transforming local potential to a uniform radial grid of dG=0.02 with 1148 points.
  Transforming nonlocal projectors to a uniform radial grid of dG=0.02 with 432 points.
  Transforming density augmentations to a uniform radial grid of dG=0.02 with 1148 points.
  Transforming atomic orbitals to a uniform radial grid of dG=0.02 with 432 points.
  Core radius for overlap checks: 1.70 bohrs.

Initialized 2 species with 35 total atoms.

Folded 1 k-points by 1x1x1 to 1 k-points.

---------- Setting up k-points, bands, fillings ----------
No reducable k-points. 
Computing the number of bands and number of electrons
Calculating initial fillings.
nElectrons: 186.000000   nBands: 93   nStates: 1

----- Setting up reduced wavefunction bases (one per k-point) -----
average nbasis = 534137.000 , ideal nbasis = 534009.526

---------- Setting up ewald sum ----------
Optimum gaussian width for ewald sums = 11.028981 bohr.
Real space sum over 729 unit cells with max indices [  4  4  4  ]
Reciprocal space sum over 4913 terms with max indices [  8  8  8  ]

---------- Allocating electronic variables ----------
Skipped wave function initialization.


---- Citations for features of the code used in this run ----

   Software package:
      R. Sundararaman, K. Letchworth-Weaver, K.A. Schwarz, D. Gunceler, Y. Ozhabes and T.A. Arias, 'JDFTx: software for joint density-functional theory', SoftwareX 6, 278 (2017)

   gga-PBE exchange-correlation functional:
      J.P. Perdew, K. Burke and M. Ernzerhof, Phys. Rev. Lett. 77, 3865 (1996)

   Pseudopotentials:
      KF Garrity, JW Bennett, KM Rabe and D Vanderbilt, Comput. Mater. Sci. 81, 446 (2014)

   Total energy minimization:
      T.A. Arias, M.C. Payne and J.D. Joannopoulos, Phys. Rev. Lett. 69, 1077 (1992)

This list may not be complete. Please suggest additional citations or
report any other bugs at https://github.com/shankar1729/jdftx/issues

Dry run successful: commands are valid and initialization succeeded.
End date and time: Sat Jun  5 22:59:40 2021  (Duration: 0-0:00:07.08)
Done!
